test/sigridcitest.py

```diff
@@ -140,6 +141,15 @@ def testForceLowerCaseForCustomerAndSystemName(self):
     def testFeedbackTemplateOnlyContainsAsciiCharacters(self):
         with open("sigridci/sigridci-feedback-template.html", mode="r", encoding="ascii") as templateRef:
             template = templateRef.read()
+            
+    def testOnlyThrowExceptionForClientError(self):
+        args = types.SimpleNamespace(partner="sig", customer="Aap", system="NOOT", sigridurl="")
+        apiClient = SigridApiClient(args)
+        apiClient.processHttpError(urllib.error.HTTPError("http://www.sig.eu", 404, "", {}, None))
+        apiClient.processHttpError(urllib.error.HTTPError("http://www.sig.eu", 502, "", {}, None))
```

Comment:
Maybe split this in 2 tests where one expects the Exception (client error) and the other (server error) does not?